alvr/server/build.rs

```diff
@@ -180,10 +180,16 @@ fn main() {
         .write_to_file(out_dir.join("bindings.rs"))
         .unwrap();
 
-    if platform_name != "macos" {
+    if platform_name != "windows" {
```

Comment:
here just put == "linux". these directive are not really needed on macos, especially because macos has even different prefix/suffixes for libraries, so they will not get picked up by cargo anyway.